src/qri/search.ts

```diff
@@ -52,37 +52,38 @@ export interface SearchResult extends Dataset {
 }
 
 export function NewSearchResult(d: Record<string,any>): SearchResult {
-  const ds = NewDataset(d)
+  const dv = d.value
+  const ds = NewDataset(dv)
   let stats
-  let followStats
-  let issueStats
+  let followStats = {}
```

Comment:
core does not supply follow or issue stats and fails calls bellow this.

Reply:
interesting. ok we may need to adjust frontend code to not require these be defined